src/Components/Search/__tests__/SearchBar.test.tsx

```diff
@@ -112,3 +115,57 @@ describe("SearchBar", () => {
     expect(component.html()).toContain("<strong>Perc</strong>y Z")
   })
 })
+
+describe("getSearchTerm", () => {
+  function buildLocationWithQueryString(queryString: string): Location {
+    return {
+      ancestorOrigins: undefined,
+      host: undefined,
+      hostname: undefined,
+      href: undefined,
+      origin: undefined,
+      port: undefined,
+      protocol: undefined,
+      assign: undefined,
+      hash: undefined,
+      pathname: undefined,
+      reload: undefined,
+      replace: undefined,
+      search: queryString,
+    }
+  }
+
+  it("returns empty string if there is no term", () => {
+    const location = buildLocationWithQueryString(undefined)
+
+    const result = getSearchTerm(location)
+
+    expect(result).toEqual("")
+  })
+
+  it("returns empty string if there is an empty term", () => {
+    const location = buildLocationWithQueryString("")
+
+    const result = getSearchTerm(location)
+
+    expect(result).toEqual("")
+  })
+
+  it("returns the term if there is a single one", () => {
+    const location = buildLocationWithQueryString("?term=monkey")
+
+    const result = getSearchTerm(location)
+
+    expect(result).toEqual("monkey")
+  })
+
+  it("returns the first term if there are multiple", () => {
+    // This is not really a concern in real-life, but TypeScript says
```

Comment:
Ah cool, was wondering with the array check but this makes sense, someone could do `?term[]=...&term[]=...` and it might get parsed as an array.